docs: Update configuration.asciidoc (#1015)
elastic/apm-agent-rum-js: docs/configuration.asciidoc

Set the verbosity level for the agent.

> This does not have any influence on the types of errors that are sent to the APM Server. Use this option to report an issue with the agent to us.

Reviewer: ```suggestion
This does not have any influence on the types of errors that are sent to the APM Server. This option is useful when you want to report an issue with the agent to us.
```

* *Type:* Boolean
* *Default:* `true`

> The distributed tracing is enabled by default, but this configuration can be used to disable it.

Reviewer: ```suggestion
Distributed tracing is enabled by default. Use this option to disable it.
```

* *Default:* `[]`

This option can be set to an array containing one or more Strings or RegExp objects and determines which origins should be monitored as part of distributed tracing.
This option is consulted when the agent is about to add the distributed tracing HTTP header (`traceparent`) to a request.

> Please note that each item in the array should be a valid URL containing the origin (other parts of the url are ignored) or a RegExp object. If an item in the array is a string, an exact match will be performed. If it's a RegExp object, its test function will be called with the request origin. 

Author: String is inconsistently capitalized. Should it always be capitalized?

When distributed tracing is enabled, this option can be used to propagate the https://www.w3.org/TR/trace-context/#tracestate-header[tracestate]
HTTP header to the configured origins. Before enabling this flag, make sure to change your <<server-configuration, server configuration>> to avoid

> Cross-Origin Resource Sharing errors. 

Author: Should cross origin be capitalized? Or resource sharing?

[[events-limit]]
==== `eventsLimit`

> The agent can only send up to `80` events every `60000` milliseconds (one minute) by default.

Reviewer: ```suggestion
By default, the agent can only send up to `80` events every `60000` milliseconds (one minute).
```

* *Type:* Number
* *Default:* `1.0`

> A number between `0.0` and `1.0` that specifies the sample rate of transactions. By default all transactions are sampled.

Reviewer: ```suggestion
A number between `0.0` and `1.0` that specifies the sample rate of transactions. By default, all transactions are sampled.
```
